### PR TITLE
Bugfixes according to Trello list

### PR DESCRIPTION
**Pull request by manue1**

### CODE REVIEW

File: shared/components/graphs/graphs.component.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class GraphsComponent extends Panel {
   }
 
   shouldShowTotal(){
-    if (this.current_route_name === 'GetStarted' || this.current_route_name === 'Footprint') {
+    if (this.current_route_name === 'GetStarted' || this.current_route_name === 'Footprint' || this.current_route_name === 'TakeAction') {
       return false;
```
Comment (erichulburd) [inline shared/components/graphs/graphs.component.js]: cosmetic suggestion:

``` js
  return !(this.current_route_name === 'GetStarted' || 
    this.current_route_name === 'Footprint' || 
    this.current_route_name === 'TakeAction')
```

File: shared/components/take_action/action/action.component.js [hunk 2]
```diff
@@ -113,6 +113,9 @@ class ActionComponent extends Translatable {
   setInputState(id){
     let take_action = this,
     footprint = take_action.state_manager.state.user_footprint;
+    // console.log('footprint', footprint);
+    // console.log('id', id);
+    // console.log('footprint[id]', footprint[id]);
     return footprint[id]
   }
```
Comment (erichulburd) [inline shared/components/take_action/action/action.component.js:117]: avoid committing logs and commented out code (use git history when needed).

File: shared/components/travel/travel.component.js [hunk 3]
```diff
@@ -21,17 +21,18 @@ class TravelComponent extends Panel {
...
 
   get vehicles(){
     return this.state.vehicles;
   }
 
+  get amount_of_vehicles(){
+    return parseInt(this.userApiValue('input_footprint_transportation_num_vehicles'));
+  }
+
   get vehicles_maxed(){
```
Comment (erichulburd) [inline shared/components/travel/travel.component.js:36]: ok good catch here.

I'd name it `number_of_vehicles` - a little easier to read.

File: shared/lib/state_manager/state_manager.js [hunk 4]
```diff
@@ -259,12 +260,16 @@ export default class StateManager {
...
     state_manager.state.user_footprint = {};
+    state_manager.state.location_reset = true;
+    state_manager.state.display_location = '';
+
     return state_manager.updateDefaults().then(() => {
-      // User footprint reset!
+      // Footprint reset!
       state_manager.syncLayout().then(() => {
       });
     })
```
Comment (erichulburd) [inline shared/lib/state_manager/state_manager.js:270]: shouldn't `state_manager.syncLayout()` be sufficient?

**Review by erichulburd — APPROVED** [review]
a few cosmetic suggestions.

File: shared/lib/state_manager/state_manager.js (hunk 4 above)
Comment (manue1) [reply shared/lib/state_manager/state_manager.js:270]: yes, you're definitely right, must be leftovers from having logs in there.

**Pull request merged**